client/state/selectors/is-mapped-domain-site.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getRawSite } from 'state/sites/selectors';
+
+/**
+ * Returns true if site is a mapped domain site, false if the site is not,
+ * or null if the site is unknown.
+ *
+ * @param  {Object}   state  Global state tree
+ * @param  {Number}   siteId Site ID
+ * @return {?Boolean}        Whether site is a mapped domain site
```

Comment:
No need to add the extra white space for alignment in the JSDoc comments here and elsewhere in the patch.

See https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#comments for an example.